generators/python/src/fern_python/generators/sdk/sdk_generator.py

```diff
@@ -285,10 +285,13 @@ def run(
                         project=project,
                         generated_root_client=generated_root_client,
                     )
-                except Exception:
+                except Exception as e:
                     generator_exec_wrapper.send_update(
                         GeneratorUpdate.factory.log(
-                            LogUpdate(level=LogLevel.DEBUG, message=f"Failed to generate README.md; this is OK")
+                            LogUpdate(
+                                level=LogLevel.DEBUG,
+                                message=f"Failed to generate README.md with exception {e}; this is OK",
```

Comment:
Up to make this user facing message a bit nicer: 

`"Failed to generate README.md..."`

Reply:
Improved.